psga/config/psga-schema-sars-cov-2.yaml

```diff
@@ -1,399 +1,316 @@
   protocols:
-
     # ARTIC_V1
     - name: illumina-sars-cov-2-articv1
       kit:
-        name: ARTIC_V1
-        slug: articv1
+        name: ARTIC V1
+        slug: ARTIC_V1
       sequencer_technology:
-        name: illumina
+        name: Illumina
         slug: illumina
       pathogen:
-        name: Sars-CoV-2
+        name: SARS CoV 2
         slug: sars-cov-2
 
     - name: ont-sars-cov-2-articv1
       kit:
-        name: ARTIC_V1
-        slug: articv1
+        name: ARTIC V1
+        slug: ARTIC_V1
       sequencer_technology:
-        name: ont
+        name: ONT
         slug: ont
       pathogen:
-        name: Sars-CoV-2
+        name: SARS CoV 2
         slug: sars-cov-2
 
     # ARTIC_V2
     - name: illumina-sars-cov-2-articv2
       kit:
-        name: ARTIC_V2
-        slug: articv2
+        name: ARTIC V2
+        slug: ARTIC_V2
       sequencer_technology:
-        name: illumina
+        name: Illumina
         slug: illumina
       pathogen:
-        name: Sars-CoV-2
+        name: SARS CoV 2
         slug: sars-cov-2
 
     - name: ont-sars-cov-2-articv2
       kit:
-        name: ARTIC_V2
-        slug: articv2
+        name: ARTIC V2
+        slug: ARTIC_V2
       sequencer_technology:
-        name: ont
+        name: ONT
         slug: ont
       pathogen:
-        name: Sars-CoV-2
+        name: SARS CoV 2
         slug: sars-cov-2
 
-    # ARTIC_V3
+    # ARTIC V3
     - name: illumina-sars-cov-2-articv3
       kit:
-        name: ARTIC_V3
-        slug: articv3
+        name: ARTIC V3
+        slug: ARTIC_V3
       sequencer_technology:
-        name: illumina
+        name: Illumina
         slug: illumina
       pathogen:
-        name: Sars-CoV-2
+        name: SARS CoV 2
         slug: sars-cov-2
 
     - name: ont-sars-cov-2-articv3
       kit:
-        name: ARTIC_V3
-        slug: articv3
+        name: ARTIC V3
+        slug: ARTIC_V3
       sequencer_technology:
-        name: ont
+        name: ONT
         slug: ont
       pathogen:
-        name: Sars-CoV-2
+        name: SARS CoV 2
         slug: sars-cov-2
 
-    # ARTIC_V4
+    # ARTIC V4
     - name: illumina-sars-cov-2-articv4
       kit:
-        name: ARTIC_V4
-        slug: articv4
+        name: ARTIC V4
+        slug: ARTIC_V4
       sequencer_technology:
-        name: illumina
+        name: Illumina
         slug: illumina
       pathogen:
-        name: Sars-CoV-2
+        name: SARS CoV 2
         slug: sars-cov-2
 
     - name: ont-sars-cov-2-articv4
       kit:
-        name: ARTIC_V4
-        slug: articv4
+        name: ARTIC V4
+        slug: ARTIC_V4
       sequencer_technology:
-        name: ont
+        name: ONT
         slug: ont
       pathogen:
-        name: Sars-CoV-2
+        name: SARS CoV 2
         slug: sars-cov-2
 
-    # ARTIC_V4-1
+    # ARTIC V4.1
     - name: illumina-sars-cov-2-articv4-1
       kit:
-        name: ARTIC_V4-1
-        slug: articv4-1
+        name: ARTIC V4.1
+        slug: ARTIC_V4-1
```

Comment:
I don't know how these are used, so my comment is likely pointless. 
If `name` is used by the pipeline, it should be ARTIC_V4-1 .

Reply:
slug is used by the pipeline, name is just the human readable format